README.md

```diff
@@ -300,15 +300,16 @@ You can then use the instance to call methods that perform payment operations. R
 
 `ForageTerminalSDK()` is the entry point to the Forage Terminal SDK. A `ForageTerminalSDK` instance interacts with the Forage API.
 
-To create a `ForageTerminalSDK` instance, call the constructor, passing the `posTerminalId` as the only parameter, as in the following snippet:
+To create a `ForageTerminalSDK` instance, call the constructor, passing the `posTerminalId` and a `sessionToken` as the only parameters, as in the following snippet:
 
 ```kotlin
-val forage = ForageTerminalSDK(posTerminalId)
+val forage = ForageTerminalSDK(posTerminalId, sessionToken)
 ```
 
 #### `ForageTerminalSDK()` parameters
 
 - `posTerminalId` (_required_): A string that uniquely identifies the POS Terminal used for a transaction.
+- `sessionToken` (_required_): A session token which will authenticate the POS Terminal against Forage's key management service.
```

Comment:
Thank you so much for doing this! You didn't miss anything. I'm just suggesting super nits that to this line 312 that: 

- Mirror the description Danilo and I wrote in Kdoc comments this week 
- Note that it's short-lived, link to the docs 

A short-lived token that authenticates the POS Terminal against Forage's key management service.  To create one, send a server-side `POST` from your backend to the [`/session_token/`](https://docs.joinforage.app/reference/create-session-token) endpoint.